static/src/javascripts/projects/commercial/modules/hosted/video.js

```diff
@@ -73,6 +78,22 @@ define([
 
     function init() {
         return new Promise(function (resolve) {
+            var $youtubeIframe = $('.js-hosted-youtube-video');
+            $youtubeIframe.each(function(el){
```

Comment:
is the `.each` needed as there's only one element being added to the DOM with this class